policyengine_us/variables/household/income/household/in_poverty.py

```diff
@@ -0,0 +1,10 @@
+from policyengine_us.model_api import *
+
+
+class in_poverty(Variable):
+    label = "in poverty"
+    documentation = "Whether household is in poverty"
+    entity = Household
+    definition_period = YEAR
+    value_type = bool
+    adds = ["spm_unit_is_in_spm_poverty"]
```

Comment:
BTW why do we need this and the empty `poverty_gap` variable @nikhilwoodruff?

Reply:
Nikhil to re-do in subsequent PRs to enable deep poverty and poverty gap across countries